package.json

```diff
@@ -93,7 +93,8 @@
     "secretcheck"
   ],
   "resolutions": {
-    "lodash": ">=4.17.11"
+    "lodash": ">=4.17.11",
+    "pubsweet-client/graphql": "^14.2.1"
```

Comment:
you can also just say `graphql`? You really want to use a single version of it everywhere.

Reply:
This makes sense as it'll break again if anything else includes a different version.